effekt/jvm/src/main/scala/effekt/Runner.scala

```diff
@@ -54,8 +54,26 @@ trait Runner[Executable] {
   /**
    * Runs the executable (e.g. the main file) by calling the build function.
    */
-  def eval(executable: Executable)(using Context): Unit =
-    exec(build(executable))
+  def eval(executable: Executable)(using C: Context): Unit = {
+    val execFile = build(executable)
+
+    val exitCode = Process(execFile).run(new ProcessLogger {
+
+      override def out(s: => String): Unit = {
+        C.config.output().emitln(s)
+        System.out.flush()
```

Comment:
Do we need to flush each time?

Reply:
We can probably drop this; it is left over from trying to get the ordering right when printing not-full-lines.
`emitln` should flush anyway.